N20: Release

ozkriff opened the pull request:
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/4d55e74ad1e5a49680bc18037823f0a6493435b9): https://rust-gamedev-preview.netlify.app/posts/newsletter-020

Everyone is invited to review!

:timer_clock: Merging this in ~4/5 hours if there're no objections.

_Part of #531_

djkoloski commented on content/posts/newsletter-020/index.md:
```diff
@@ -1060,14 +1036,14 @@ Version 0.5 is hot off the presses and rolls up features from the 0.4
-- `const_generics` are now enabled by default
-- Helper functions have been added to make getting root objects easier
-- Several bugfixes and performance improvements
+  [Infallible][rkyv-Infallible] were introduced to improve ergonomics.
```
were => was

ozkriff replied:
:thinking: Are you sure? "Helper **types** like AlignedVec and Infallible **were** introduced to improve ergonomics"

djkoloski replied:
My bad, there's nothing to fix here. In my morning grog I missed the previous markdown line.

djkoloski reviewed (changes requested):
I used the wrong plurality, oops! Everything looks really good otherwise. 🙂

kvark commented:
> e.g. posting issues on our github.

This contradicts the guidelines, I think. Shouldn't be written as "our".

> Station Iapetus  

Has both the image as a link, and the link in the image caption. Should the caption just say "click on the image to see the video", or the image be non-link instead?

> Emerald

The image here doesn't need to be so big, since it's just a logo, it doesn't show any actual rendering.

Honestly, this entry is also a bit fishy... The news for Emerald are just these two tiny lines:
>    rendering to textures
    WASM game sample.

While everything else is self-promotion and general information. I think the newsletter should put a stronger focus on the "news" part, and prevent parties from just using it to remind about themselves. I.e. something like "your news need to be at least as big as the general description" rule.

Similar concern could be applied to "Quinn" entry, although to a much lesser degree.

The "bevy" entry style is what I find to be the best, and every entry should try to follow the format.

In "wgpu" entry, something goes wrong about the image caption, it's not centered.

> (such as #[allow(unused_attributes)])

Would be nice to codify this part (i.e. with "`")

> based on the wgpu framework

nit: `wgpu` is a library, not a framework

ozkriff commented:
@kvark addressed most of the suggestions, thanks!

> The news for Emerald are just these two tiny lines:
> While everything else is self-promotion and general information. I think the newsletter should put a stronger focus on the "news" part, and prevent parties from just using it to remind about themselves. I.e. something like "your news need to be at least as big as the general description" rule.

This is the first newsletter issue that features Emerald's entry. It's ok that it focuses on a general overview.

> Similar concern could be applied to "Quinn" entry, although to a much lesser degree.

Same with Quinn.

ozkriff commented:
:timer_clock:  Merging in 15 minutes.

kvark commented:
Ok cool, I didn't realise this is the introduction for these libraries :)

> On Apr 10, 2021, at 11:31, Andrey Lesnikóv ***@***.***> wrote:
> 
> ﻿
> Merged #583 into source.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
